### DISCUSSION TITLE
Big up's - Kudos - and shout out to Buster!

### DISCUSSION THREAD

**Discussion by marekdziem** [AzuraCast/AzuraCast#6040] [General]
I just have to say WOW! THANSK YOU! and YEH YEH YEH! to the amazing dev machine guy:

**Buster Neece!!**

@BusterNeece - a most sincere **Thank you** for all your hard work and effort on Azuracast - you totally rock!!!

..and how can I help out?
:)
marek

**Comment by BusterNeece** [comment]
Thank you for the kind words!

If you want to help keep a roof over my head so I can keep working hard on the project, visit https://donate.azuracast.com/ for options on how to provide financial support to me and the project.

We also gladly accept contributions in the form of translations, documentation updates, and pull requests for code.

**Comment by ivellios1988** [comment]
Można pomóc na wiele sposobów. Na przykład tłumacząc brakujące elementy AzuraCast na polski albo dowolny inny język, który znasz a który jest dostępny na koncie Bustera w Crowdin.

Możesz też wysłać jakiś datek albo zacząć wspieranie AzuraCast przez Patreon.

Wreszcie możesz promować AzuraCast w różnych miejscach typu fora czy grupy facebookowe o tematyce radiowej, czy choćby podsuwać streamerom i firmom hostingowym AzuraCast jako świetną alternatywę dla Centova Cast. Zresztą, AzuraCast potrafi więcej od swojego drogiego konkurenta! A przy tym ma świetnie zrobiony responsywny design (używanie Centova Cast na smartfonie to katorga...).

Możesz też promować AzuraCast podczas jakichś streamów na YouTube, w postach na Wykopie, Reddit'cie, itd.

**Comment by marekdziem** [comment]
Czesc! 

Ja jestem urodzoni w RPA I nie mowiem za dobrze po polsku – ale mogem spruwowac! 

 

I’m a first generation south African born to polish immigrants and although I can speak and to a lesser extent read and write in polish, my command of the language may be insufficient to rely on for translating!

But I am happy to try with the help of google translate of course!

 

How do I go about helping on the translations?

marek
